### ISSUE TITLE
Draft section 3.1 Rule Outline

### ISSUE THREAD

**Issue by moekraft** [w3c/wcag-act#28]
Editor note: This sections gives a broad outline of parts make up an ACT Rule. We’ll go into further detail in sections below. At this point it is important to show the reader the big picture

**Comment by moekraft** [comment]
I'm looking to Auto WCAG for inspiration. I found the [Rule template](https://github.com/WilcoFiers/auto-wcag/blob/f611f774619fa497053710793d7298c10ba42977/pages/rule-template.md). 

Questions/Comments:

1. Do we want to have the rule outline section be as detailed as the Auto WCAG rule template with descriptions/details or keep it simple like the [empty rule template.](https://github.com/WilcoFiers/auto-wcag/blob/f611f774619fa497053710793d7298c10ba42977/pages/rule-template-empty.md).

2. I assume we want to be careful not to tie the rule outline to WCAG SC. I will work to make it more generic.

3. Should we not have a Pass/Fail or Result section in the rule?

**Comment by moekraft** [comment]
Pull request #43 has been merged.
